### PR TITLE
Fix JDK naming and cleanup in ant build

### PR DESCRIPTION
**Pull request by sampottinger**
In response to processing#6, ensure removal of temporary archive expansions of JFX and JDKs, fix .gitignore for those downloads, and switch to the new JDK naming scheme.

### CODE REVIEW

File: build/.gitignore [hunk 1]
```diff
@@ -1,6 +1,12 @@
 work
 javadoc
 
-macosx/jdk-*
-macosx/javafx-sdk-11.0.2
-macosx/jfx-11.0.2.zip
+macosx/javafx-sdk-*
```
Comment (sampottinger) [inline build/.gitignore:4]: Up to you if you want these fully spelled out or collapsed with the leading wildcard. Let me know!

Comment (benfry) [reply build/.gitignore:4]: Wildcard is fine; `jfx` and `javafx-sdk` are different animals?

Comment (sampottinger) [reply build/.gitignore:4]: The name of the folder in the JFX distribution zip is `javafx-sdk`.

Comment (sampottinger) [reply build/.gitignore:4]: I can change the name of the downloaded file but was trying to keep format the same between jdk and jfx. Let me know if you want me to switch!

**Pull request merged**